force-app/main/default/objects/WorkOrder/fields/HOT_StatusChanged__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>HOT_StatusChanged__c</fullName>
```

Comment:
Denne har kanskje litt for likt navn med HOT_IsUpdateStatus__c? Bør det nye feltet kanskje hete noe med SMS og/eller varsel?

Reply:
Forsalg: HOT_SendSMSNotificationOnStatusChanged__c, eller noe liknende